package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "VerilogHDL",
```

Comment:
Any reason to change the extension name?

Reply:
The "name" field should be lowercase pony as described in https://code.visualstudio.com/docs/extensionAPI/extension-manifest

The old name should be moved to "displayName"

Reply:
Ok.